base_layer/wallet/src/output_manager_service/handle.rs

```diff
@@ -106,6 +111,10 @@ pub enum OutputManagerRequest {
     ApplyEncryption(Box<Aes256Gcm>),
     RemoveEncryption,
     GetPublicRewindKeys,
+    CalculateRecoveryByte {
```

Comment:
This seems strange to expose at this level. What is the use case for someone to only call this method without an output?

Reply:
This was needed for the wallet FFI method `pub unsafe extern "C" fn wallet_import_utxo`as it could accept a null pointer for OutputFeatures and then assign default features, in which case the recovery byte had to be set correctly to pass the recovery byte validation further down the line. Coming to think of it, all input data should be coherent, so changing the FFI method to return an error in case of null pointer for OutputFeatures.

Reply:
This is now only used for unit tests with a `ToDo` comment.